src/hooks/useNotifications.ts

```diff
@@ -60,10 +60,9 @@ export default async function useNotifications(uuid: string = '') {
               window.location.href = notification.resource?.redirect_to_url;
               break;
             case NotificationsTypes.ACCOUNT_LOGOUT:
+              accountController.logout();
```

Comment:
Should we wait for the logout first?

Reply:
https://www.notion.so/jwplayer/Viewers-are-able-to-subvert-the-Simultaneous-login-limit-baf6baf8ba9946978ea573d871644279

Reply:
Because there was a problem with the simultaneous log in, it shows the log in component but it didn't logs out the user.

Reply:
I mean using `await` for logout.

Reply:
If i use await, i have problem with showing the log in modal it doesn't show up.Do you have another suggestion for this bug ?

Reply:
d386c9ccfb350655f7b4219e9d22563fb9655547 @AntonLantukh